web/styles/global.css

```diff
@@ -16,7 +16,7 @@ body {
   min-height: 100%;
   /* Add the GEM as background */
   /* background: url(../public/gem3.svg) no-repeat top left; */
-  background: url(../public/background.svg) no-repeat center center fixed;
+  background: #ffffff;
```

Comment:
This makes some of the other pages look very strange. Could you maybe change it back for the main page at the minimum?

Reply:
Reverted background changes other pages and only applied `background: #ffffff` for nlAugmenter page.